### PR TITLE
Apply bzip2 shared library fix and rerender recipe - Python 2.7

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

**Comment by jakirkham** [comment]
Oops, had just added PR ( https://github.com/conda-forge/python-feedstock/pull/140 ). Feel free to close it though if you would rather. Thanks for doing these.

**Comment by jjhelmus** [comment]
@jakirkham Just saw that as well.  Thanks for putting together the bzip2 fix, I borrow it for these PRs.

**Comment by jjhelmus** [comment]
I do not believe the failing Windows build is the result of the rerender, rather `%PYTHON%` is not defined when building python, conda/conda-build#1960.  Let me think about the best method for solving this.  It might be necessary to provide a full path to the newly built Python interpreter in the `bld.bat` script.

**Comment by jjhelmus** [comment]
I think this is good to go.  Anyone care to review the changes?

File: recipe/bld.bat [hunk 1]
```diff
@@ -150,9 +151,9 @@ if errorlevel 1 exit 1
 
 REM bytecode compile the standard library
 
-%PYTHON% -Wi %STDLIB_DIR%\compileall.py -f -q -x "bad_coding|badsyntax|py3_" %STDLIB_DIR%
+%PREFIX%\python.exe -Wi %PREFIX%\Lib\compileall.py -f -q -x "bad_coding|badsyntax|py3_" %PREFIX%\Lib
```
Comment (jakirkham) [inline recipe/bld.bat]: After looking everything, this change seems fine as these paths all end up being the same in the end.

**Pull request merged**